docs: DOC-1133 (#2582)
spectrocloud/librarium: docs/docs-content/release-notes.md

  through Palette CLI will be eligible for a cluster profile update. We recommend you review the
  [Upgrade a PCG](./clusters/pcg/manage-pcg/pcg-upgrade.md) guide to learn more about updating a PCG.

> - Self-hosted Palette instances will now use Kubernetes version 1.27.9. The new version of Kubernetes will cause node

Reviewer: ```suggestion
- Self-hosted Palette instances now use Kubernetes version 1.27.9. This new version of Kubernetes will cause node
```

> - Self-hosted Palette instances now use Kubernetes version 1.27.9. This new version of Kubernetes will cause node repave

Reviewer 2: 1.27.11 is the k8s version now

Author: Thank you @tiwarisumit3

> - Self-hosted Palette instances now use Kubernetes version 1.27.9. This new version of Kubernetes will cause node repave

Author: ```suggestion
- Self-hosted Palette instances now use Kubernetes version 1.27.11. This new version of Kubernetes will cause node repave
```

  [VMware Resources Remain After Cluster Deletion](./troubleshooting/pcg.md#scenario---vmware-resources-remain-after-cluster-deletion)
  troubleshooting guide for resolution steps.

  <!-- prettier-ignore -->

- In a VMware environment, self-hosted Palette instances do not receive a unique cluster ID when deployed, which can

>   cause issues during a node repave event, such as a Kubernetes version upgrade. Specifically, Persistent Volume (PV)

Reviewer: ```suggestion
  cause issues during a node repave event, such as a Kubernetes version upgrade. Specifically, Persistent Volumes (PVs)
```

>   and Persistent Volume Claim (PVC) will experience start problems due to the lack of a unique cluster ID. To resolve

Reviewer: ```suggestion
  and Persistent Volume Claims (PVCs) will experience start problems due to the lack of a unique cluster ID. To resolve
```

  that are not compliant are reported with either a failed or unknown status. Refer to the
  [FIPS Validate](./palette-cli/commands/fips-validate.md) guide to learn more about the command.

> - VerteX instances now use Kubernetes version 1.27.9. This new version of Kubernetes will cause node repave events

Author: ```suggestion
- VerteX instances now use Kubernetes version 1.27.11. This new version of Kubernetes will cause node repave events
```

> - VerteX instances will now use Kubernetes version 1.27.9. The new version of Kubernetes will cause node repave events

Reviewer: ```suggestion
- VerteX instances now use Kubernetes version 1.27.9. This new version of Kubernetes will cause node repave events
```
